mod/item.php

```diff
@@ -331,13 +326,13 @@ function item_post(App $a) {
 			$private           = $parent_item['private'];
 		}
 
-		$pubmail_enable    = ((x($_REQUEST, 'pubmail_enable') && intval($_REQUEST['pubmail_enable']) && (! $private)) ? 1 : 0);
+		$pubmail_enable    = ((x($_REQUEST, 'pubmail_enable') && intval($_REQUEST['pubmail_enable']) && !$private) ? 1 : 0);
```

Comment:
This can be simplified to `$pubmail_enable    = (x($_REQUEST, 'pubmail_enable') && !$private);`.